safe_transaction_service/history/tasks.py

```diff
@@ -270,7 +270,7 @@ def process_decoded_internal_txs_task(self) -> Optional[int]:
 
 
 @app.shared_task(bind=True, soft_time_limit=SOFT_TIMEOUT, time_limit=LOCK_TIMEOUT)
-def reindex_last_hours_task(self, hours: int = 2) -> Optional[int]:
+def reindex_last_hours_mastercopies_task(self, hours: int = 2) -> Optional[int]:
```

Comment:
I think name is a little off (my fault(), as we are renaming it maybe `reindex_mastercopies_last_hours_task` would be better

Reply:
Sounds good